test/e2e/operator_test.go

```diff
@@ -255,7 +286,7 @@ func setImage(deployment *appsv1.Deployment, image string) {
 }
 
 func checkCurrentDNSImage(t *testing.T, cl client.Client, expectedImage string) error {
-	err := wait.PollImmediate(1*time.Second, 7*time.Minute, func() (bool, error) {
+	err := wait.PollImmediate(1*time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
What was the motivation for both the timeout changes? CI runs? other?

Reply:
I observed `checkCurrentDNSImage` taking close to the 7 minute timeout when I ran the test on my own cluster. Just added some cushion. Dropped the 5 minute timeout for fetching the deployment since that just seems excessive. 

I can revert these changes if you'd prefer.

Reply:
> I can revert these changes if you'd prefer.

If they're currently OK and passing in CI I'd be inclined to revert.

Reply:
reverted.